atm/Sidorchuk/task15/ATM/BankAccount.java

```diff
@@ -0,0 +1,147 @@
+package task15.ATM;
+
+import task15.ATM.money.Currency;
+import java.util.Date;
+import java.util.Random;
+
+public class BankAccount {
+
+   private class Account{
+        private Currency typeOfAccount;
+        private int balance;
+
+        public Account(Currency typeOfAccount, int balance) {
+            this.typeOfAccount = typeOfAccount;
+            this.balance = balance;
+        }
+
+        public Currency getTypeOfAccount() {
+            return typeOfAccount;
+        }
+
+        public int getBalance() {
+            return balance;
+        }
+
+        public void setBalance(int balance) {
+            this.balance = balance;
+        }
+    }
+   private Account acccount1;
```

Comment:
Lots of duplicate code. you can use some Collection instead of creating account1, account2, ...